ci/rpm7/build.sh

```diff
@@ -0,0 +1,371 @@
+#!/bin/bash
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# Author: Dmitrii Demin
+
+url_src='https://github.com/telefonicaid/fiware-orion'
+url_dst='https://nexus.lab.fiware.org/repository/el'
+releasever=7
+basearch='x86_64'
+
+function _usage()
+{
+  echo -n "Usage: build [options]
+  Options:
+    -h   --help          show help
+    -b   --branch        specify branch/tag to build, if not specified - the source from /opt/fiware-orion will be used
+    -s   --source        specify repository to clone, if not specified - default repo (https://github.com/telefonicaid/fiware-orion) will be used
+    -p   --path          specify path to use as home, if not specified - /opt/fiware-orion will be used
+    -M   --make          cmake/make, stage (unit/functional) should be specified
+    -I   --install       make install
+    -c   --compliance    run file_compliance, payload and style checks
+    -u   --unit          run unit tests
+    -f   --functional    run functional tests
+    -n   --nightly       build rpm nightly
+    -r   --release       build rpm release
+    -t   --testing       build rpm testing
+    -U   --upload        upload rpm, REPO_USER and REPO_PASSWORD ENV variables should be provided
+    -F   --fix           execute fix for jenkins and travis (disable ipv6 test)
+    -E   --execute       run (rerun) test stand with 2 orions
+    -D   --db            start mongodb
+    -S   --show          show the list of necessary commands that should be executed before starting functional tests manually
+
+  Examples:
+    build -M unit -IDu -b master   clone from master, make (for unit testing), make install, run mongo, execute unit tests
+    build -M functional -IDFf      get source from mounted folder, make (for functional testing), make install, run mongo, execute FIX, functional test
+"
+  exit 0
+}
+
+function _fix_jenkins()
+{
+    # FIXME: ipv6 should be enabled for testing
```

Comment:
I'd try to be more precise in the FIXME message. Suggestion:

> FIXME: ipv4_ipv6_both.test is disabled as we have been not able to make it work in the CI environment. This should be fixed.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3192/commits/b3b5fece8267bcdc815ba091ff7f2cc626c3cc91